### PR TITLE
Movies details

### PR DESCRIPTION
**Pull request by ekalac1**

### CODE REVIEW

File: MovieApp/app/src/main/java/ba/unsa/etf/rma/elza_kalac/movieapp/Fragments/LatestMoviesFragment.java [hunk 1]
```diff
@@ -68,7 +67,7 @@ public void onFailure(Call<MoviesListResponse> call, Throwable t) {
             public boolean onLoadMore(int page, int totalItemsCount) {
                 ApiInterface apiService =
                         ApiClient.getClient().create(ApiInterface.class);
-                Call<MoviesListResponse> call = apiService.getMostPopularMovies(ApiClient.API_KEY, pageNum);
+                Call<MoviesListResponse> call = apiService.getMostPopularMovies(ApiClient.API_KEY, page);
```
Comment (ekalac1) [inline MovieApp/app/src/main/java/ba/unsa/etf/rma/elza_kalac/movieapp/Fragments/LatestMoviesFragment.java]: Removed.

File: MovieApp/app/src/main/java/ba/unsa/etf/rma/elza_kalac/movieapp/Models/Movie.java [hunk 2]
```diff
@@ -168,13 +186,13 @@ public String getTitle() {
...
-
-    public String getFullPosterPath()
-    {
-        return "http://image.tmdb.org/t/p/"+"w500"+getPosterPath();
+    public double getVoteAverage() {
+        return voteAverage;
     }
 
+    public String getFullPosterPath() {
+        return "http://image.tmdb.org/t/p/" + "w500" + getPosterPath();
```
Comment (ekalac1) [inline MovieApp/app/src/main/java/ba/unsa/etf/rma/elza_kalac/movieapp/Models/Movie.java]: Done

File: MovieApp/app/src/main/java/ba/unsa/etf/rma/elza_kalac/movieapp/Fragments/LatestTvShowsFragment.java [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                Toast.makeText(getActivity().getApplicationContext(), R.string.on_failure, Toast.LENGTH_LONG).show();
+                return;
+            }
+        });
+        grid.setOnScrollListener(new EndlessScrollListener() {
+            @Override
+            public boolean onLoadMore(int page, int totalItemsCount) {
+                final ApiInterface apiService = ApiClient.getClient().create(ApiInterface.class);
+
+                Call<TvShowResponse> call = apiService.getHighestRatedTvShows(ApiClient.API_KEY, page);
```
Comment (irma-abh) [inline MovieApp/app/src/main/java/ba/unsa/etf/rma/elza_kalac/movieapp/Fragments/LatestTvShowsFragment.java]: apiService.getLatestTvShows(ApiClient.API_KEY, page);

File: MovieApp/app/src/main/java/ba/unsa/etf/rma/elza_kalac/movieapp/Models/TvShow.java [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+    public String getOriginalName() {
+        return originalName;
+    }
+
+    public void setOriginalName(String originalName) {
+        this.originalName = originalName;
+    }
+
+    public String getFullPosterPath() {return "http://image.tmdb.org/t/p/" + "w500" + getPosterPath();}
```
Comment (irma-abh) [inline MovieApp/app/src/main/java/ba/unsa/etf/rma/elza_kalac/movieapp/Models/TvShow.java]: This should be changed.

**Pull request merged**